lib/kibana/kibana/resources.py

```diff
@@ -127,6 +132,87 @@ def find_elastic(cls, **params):
         params = cls._add_internal_filter(True, params)
         return cls.find(**params)
 
+    @classmethod
+    def bulk_action(
+        cls, action: definitions.RuleBulkActions, rule_ids: Optional[List[str]] = None, query: Optional[str] = None,
+        dry_run: Optional[bool] = False, edit_object: Optional[list[definitions.RuleBulkEditActionTypes]] = None,
+        include_exceptions: Optional[bool] = False, **kwargs
+    ) -> (dict, List['RuleResource']):
+        assert not (rule_ids and query), 'Cannot provide both rule_ids and query'
```

Comment:
```suggestion
        cls, action: definitions.RuleBulkActions, rule_ids: Optional[List[str]] = None, query: Optional[str] = None,
        dry_run: Optional[bool] = False, edit_object: Optional[list[definitions.RuleBulkEditActionTypes]] = None,
        include_exceptions: Optional[bool] = False, **kwargs
    ) -> (dict, List['RuleResource']):
        assert not (rule_ids and query), 'Cannot provide both rule_ids and query'
        """Perform a bulk action on rules using the _bulk_action API."""
```

Maybe add a docstring here?